packages/beacon-node/test/unit/chain/prepareNextSlot.test.ts

```diff
@@ -20,7 +20,7 @@ describe("PrepareNextSlot scheduler", () => {
   let regenStub: MockedBeaconChain["regen"];
   let loggerStub: MockedLogger;
   let beaconProposerCacheStub: MockedBeaconChain["beaconProposerCache"];
-  let getForkStub: SpyInstance<[number], ForkName>;
+  let getForkStub: MockInstance<[number], ForkName>;
```

Comment:
What does this switch bring for this test?

Reply:
The old type was deprecated and replaced with 'MockInstance'